### ISSUE TITLE
Playlist() randomizes videos

### ISSUE THREAD

**Issue by M3Horizun** [jnsougata/Ditch-YouTubeAPI#4]
I am using this.
```py
from DYA import Playlist, Video


playlist = Playlist("https://www.youtube.com/playlist?list=PLZ9QOgK214HDaY8NfkjvTX1E-8DSOA8m9")

videos = playlist.videos()

for i in range(len(videos)):
    print(videos[i].title)
```
And it giving me random videos back. Please run the code and see for yourself.

**Comment by jnsougata** [comment]
wait for next patch

**Comment by jnsougata** [comment]
Issue has been resolved in `v3.4.8`. Please update
